### PR TITLE
updating doc for rabbitmq firehose

### PR DESCRIPTION
**Pull request by ravishrathod**
Updating doc for rabbitmq firehose as per discussion:
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/druid-user/r7emLGYrf3Q/HzgdUtjRg18J

### CODE REVIEW

File: docs/content/ingestion/firehose.md [hunk 1]
```diff
@@ -38,7 +38,58 @@ See [Examples](../tutorials/examples.html). This firehose creates a stream of ra
 #### RabbitMqFirehose
 
 This firehose ingests events from a define rabbit-mq queue.
+<br>
+**Note:** Add **amqp-client-3.2.1.jar** to lib directory of druid to use this firehose.
```
Comment (nishantmonu51) [inline docs/content/ingestion/firehose.md]: this library is part of the extension dependency, If we set the classpath properly before using the methods this should work, I believe its more like a classpath issue than the docs.

**Comment by fjy** [comment]
:+1:

**Comment by fjy** [comment]
@ravishrathod Do you mind signing the CLA at http://druid.io/community/cla.html

**Comment by ravishrathod** [comment]
@fjy Signed the CLA

**Pull request merged**